src/serac/coefficients/coefficient_extensions.cpp

```diff
@@ -11,141 +11,8 @@
 
 namespace serac::mfem_ext {
 
-mfem::Array<int> MakeTrueEssList(mfem::ParFiniteElementSpace& pfes, mfem::VectorCoefficient& c)
-{
-  mfem::Array<int> ess_tdof_list(0);
+namespace detail {
```

Comment:
was this, now empty, file left for a reason?  I can think of a few reasons for an empty file but not in this case.

Reply:
Ahh yes. I need to update this branch with what's recently merged into develop. It should disappear.